packages/components/pie-checkbox-group/src/index.ts

```diff
@@ -85,8 +90,9 @@ export class PieCheckboxGroup extends FormControlMixin(RtlMixin(LitElement)) imp
                 ?disabled=${disabled}
                 aria-describedby="${ifDefined(assistiveText ? assistiveTextId : undefined)}"
                 data-test-id="pie-checkbox-group"
+                class="c-checkbox-group ${isInline && '__inline'}"
```

Comment:
Would be a bit cleaner to use `classmap` directive here (see the [button PR](https://github.com/justeattakeaway/pie/pull/1587), which uses that)